intelmq/lib/upgrades.py

```diff
@@ -531,6 +533,21 @@ def v230_csv_parser_parameter_fix_1(defaults, runtime, harmonization, dry_run):
                 changed = True
     return changed, defaults, runtime, harmonization
 
+def v300_bots_file_removal(defaults, runtime, harmonization, dry_run):
+    """
+    Remove BOTS file
+    """
+    changed = None
+    messages = []
+    bots_file = Path(resource_filename('intelmq', 'bots/BOTS'))
```

Comment:
That will resolve to the filename in the packaging source, not the installed one. Use the deleted code of `__init__.py` (`os.path.join(intelmq.CONFIG_DIR, "BOTS")`)